mlrun/runtimes/daskjob.py

```diff
@@ -602,7 +596,7 @@ def enrich_dask_cluster(function, secrets, client_version):
         env.append(spec.extra_pip)
 
     pod_labels = get_resource_labels(function, scrape_metrics=config.scrape_metrics)
-    worker_args = ["dask-worker", "--nthreads", str(spec.nthreads)]
+    worker_args = ["dask worker", "--nthreads", str(spec.nthreads)]
```

Comment:
I wonder what is the effect of this change... could it be that it never worked properly?

Reply:
this is about to be deprecated. I will add a todo and make the change in another PR